srcCpp/PerftestTransport.cxx

```diff
@@ -49,6 +49,9 @@ PerftestTransport::PerftestTransport()
             TRANSPORT_SHMEM,
             "SHMEM",
             "dds.transport.shmem.builtin");
+
+    minimumMessageSizeMax = MESSAGE_SIZE_MAX_NOT_SET;
+    loggingString = "";
```

Comment:
Will add a comment. But basically we need to set the minimumMessageSizeMax to the maximum size of a long, so later it can be compared to the different message Size Max of the different transport and use the smallest one.

Reply:
Done.